README.md

```diff
@@ -3,30 +3,70 @@
 [![Build Status](https://travis-ci.org/OpenNMT/OpenNMT-py.svg?branch=master)](https://travis-ci.org/OpenNMT/OpenNMT-py)
 [![Run on FH](https://img.shields.io/badge/Run%20on-FloydHub-blue.svg)](https://floydhub.com/run?template=https://github.com/OpenNMT/OpenNMT-py)
 
```

Comment:
Maybe add a badge link directly to the documentation site?
Looks like many other repo do so, ref. fairseq, huggingface, etc.
Also wondering if we can set up a coverage test CI to encourage more unitests making sure the change won't break some module.

Reply:
Why not for the badge, I'll add it.
Coverage is a good idea. It may require a bit more time though. We'll add it later.